packages/ChangelogLinker/tests/ChangeTree/ChangeFactory/ChangeFactoryTest.php

```diff
@@ -57,6 +57,9 @@ public function provideDataForMessageWithoutPackage(): Iterator
     {
         yield ['[SomePackage] SomeMessage', '- [#10] [SomePackage] SomeMessage', '- [#10] SomeMessage'];
         yield ['[coding-standards] SomeMessage', '- [#10] [coding-standards] SomeMessage', '- [#10] SomeMessage'];
+        yield ['[framework] javascript stuff', '- [#10] [framework] javascript stuff', '- [#10] javascript stuff'];
+        yield ['*SomeMessage', '- [#10] \*SomeMessage', '- [#10] \*SomeMessage'];
+        yield ['*SomeMessage**', '- [#10] \*SomeMessage\*\*', '- [#10] \*SomeMessage\*\*'];
```

Comment:
This covers cases like: https://github.com/shopsys/shopsys/pull/446/commits/28950151f72fe220017263c2b7eea5b5ce8bdc2a#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR54